python/tvm/script/tir/ty.py

```diff
@@ -30,8 +30,11 @@ def evaluate(self):
         """Return an actual ir.Type Object that this Generic class wraps"""
         raise TypeError("Cannot get tvm.Type from a generic type")
 
+    def __call__(self):
+        raise NotImplementedError
```

Comment:
also add a comment explain why this is needed
```suggestion
        raise NotImplementedError()
```

Reply:
done